dhis-2/dhis-services/dhis-service-dxf2/src/main/java/org/hisp/dhis/dxf2/events/event/EventStore.java

```diff
@@ -70,6 +71,8 @@ List<Event> getEvents( EventSearchParams params, List<OrganisationUnit> organisa
 
     int getEventCount( EventSearchParams params, List<OrganisationUnit> organisationUnits );
 
+    void updateEnrollmentsLastUpdatedUserInfo( List<String> enrollmentUids, UserInfoSnapshot userInfoSnapshot );
```

Comment:
Good point, I think it would be great if you could add javadoc to this new method @lucaCambi77 

As to when to use javadoc, the short answer is always 🤔 ? Especially I think it's useful in interfaces. Even if things are straight forward, javadocs should be added. We should enforce this more :)